### ISSUE TITLE
Tags are Broken

### ISSUE THREAD

**Issue by gdurandvadas** [cloudposse/terraform-null-label#12]
Hi, I did a test of the module:
```hcl
module "eg_prod_bastion_abc_label" {
  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
  namespace  = "eg"
  stage      = "prod"
  name       = "bastion"
  attributes = ["abc"]
  delimiter  = "-"
  tags       = "${map("BusinessUnit", "ABC")}"
}

resource "aws_security_group" "eg_prod_bastion_abc" {
  name = "${module.eg_prod_bastion_abc_label.id}"
  tags = "${module.eg_prod_bastion_abc_label.tags}"

  ingress {
    from_port   = 22
    to_port     = 22
    protocol    = "tcp"
    cidr_blocks = ["0.0.0.0/0"]
  }
}
```

**output**:
```bash
✗ terraform apply
provider.aws.region
  The region where AWS operations will take place. Examples
  are us-east-1, us-west-2, etc.

  Default: us-east-1
  Enter a value: us-east-1

aws_security_group.eg_prod_bastion_abc: Creating...
  description:                           "" => "Managed by Terraform"
  egress.#:                              "" => "<computed>"
  ingress.#:                             "" => "1"
  ingress.2541437006.cidr_blocks.#:      "" => "1"
  ingress.2541437006.cidr_blocks.0:      "" => "0.0.0.0/0"
  ingress.2541437006.description:        "" => ""
  ingress.2541437006.from_port:          "" => "22"
  ingress.2541437006.ipv6_cidr_blocks.#: "" => "0"
  ingress.2541437006.protocol:           "" => "tcp"
  ingress.2541437006.security_groups.#:  "" => "0"
  ingress.2541437006.self:               "" => "false"
  ingress.2541437006.to_port:            "" => "22"
  name:                                  "" => "<computed>"
  owner_id:                              "" => "<computed>"
  revoke_rules_on_delete:                "" => "false"
  tags.%:                                "" => "<computed>"
  vpc_id:                                "" => "<computed>"
aws_security_group.eg_prod_bastion_abc: Creation complete after 6s (ID: sg-59c3e52b)

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

**checking tags of the created resource**
```json
✗ aws ec2 describe-tags --filters "Name=resource-id,Values=sg-59c3e52b"
{
    "Tags": []
}
```

**Comment by SweetOps** [comment]
Hi, sorry for long delay, that issue will be fixed in next release. At current moment you can use more older version `source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"`

**Comment by aknysh** [comment]
The issue was introduced when the `enabled` flag was added (Terraform booleans and strings were mixed up). Terraform does not recommend using booleans at all, they are not first-class citizens and are very inconsistent (not easy to reason about, even they admit it).
https://www.terraform.io/docs/configuration/variables.html#booleans

It will be fixed in this PR: https://github.com/cloudposse/terraform-null-label/pull/13

But we need to change to default = "true" in `variables.tf`, not change the tertiary operator because it will break other modules that already use  `enabled = "true"` or `enabled = "false"` (with strings).

@gdurandvadas it will be fixed today.
If you want to use the latest tag (0.3.0) right now, please add the attribute `enabled = "true" to your module (notice it's a string):

``` hcl
module "eg_prod_bastion_abc_label" {
  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
  namespace  = "eg"
  stage      = "prod"
  name       = "bastion"
  attributes = ["abc"]
  delimiter  = "-"
  tags       = "${map("BusinessUnit", "ABC")}"
  enabled    = "true"
}
```

**Comment by gdurandvadas** [comment]
@aknysh Thanks! I will be making some test today. Have an excellent day.

**Comment by aknysh** [comment]
if it's a different issue with tags (not with the `enabled` flag, please let us know, will look into it

**Comment by comeanother** [comment]
@gdurandvadas @aknysh @SweetOps 
Fixed in #13

**Comment by aknysh** [comment]
thanks @comeanother
